dynamic-simulation/dynamic-simulation-dsl/src/test/java/com/powsybl/dynamicsimulation/groovy/GroovyEventModelSupplierTest.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.powsybl.dynamicsimulation.groovy;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.io.IOException;
+import java.nio.file.FileSystem;
+import java.nio.file.Files;
+import java.util.List;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.google.common.jimfs.Configuration;
+import com.google.common.jimfs.Jimfs;
+import com.powsybl.dynamicsimulation.EventModel;
+import com.powsybl.dynamicsimulation.EventModelsSupplier;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.test.EurostagTutorialExample1Factory;
+
+/**
+ * @author Marcos de Miguel <demiguelm at aia.es>
+ */
+public class GroovyEventModelSupplierTest {
+
+    private FileSystem fileSystem;
+
+    @Before
+    public void setup() throws IOException {
+        fileSystem = Jimfs.newFileSystem(Configuration.unix());
+
+        Files.copy(getClass().getResourceAsStream("/eventModels.groovy"), fileSystem.getPath("/eventModels.groovy"));
+    }
+
+    @After
+    public void tearDown() throws IOException {
+        fileSystem.close();
+    }
+
+    @Test
+    public void test() {
+        Network network = EurostagTutorialExample1Factory.create();
+
+        List<EventModelGroovyExtension> extensions = GroovyExtension.find(EventModelGroovyExtension.class, "dummy");
+        assertEquals(1, extensions.size());
+        assertTrue(extensions.get(0) instanceof EventModelGroovyExtension);
+
+        EventModelsSupplier supplier = new GroovyEventModelsSupplier(fileSystem.getPath("/eventModels.groovy"), extensions);
+
+        List<EventModel> eventModels = supplier.get(network);
+        assertEquals(2, eventModels.size());
+
+        assertTrue(eventModels.get(0) instanceof DummyEventModel);
+        DummyEventModel dynamicModel1 = (DummyEventModel) eventModels.get(0);
```

Comment:
Here dynamicModel1 and dynamicModel2 should be eventModel1 and eventModel2

Reply:
Done